msa/black-box-tests/src/test/resources/docker-selenium.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright © 2016-2022 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '3'
+services:
+  selenium-chrome:
+    restart: always
+    image: selenium/standalone-chrome
+    ports:
+      - '4444:4444'
+      - '7900:7900'
+    shm_size: 2gb
+    environment:
+      SE_NODE_MAX_SESSIONS: 8
+      SE_NODE_OVERRIDE_MAX_SESSIONS: 'true'
+      SE_NODE_SESSION_TIMEOUT: 5000
+      SE_SCREEN_WIDTH: 1920
+      SE_SCREEN_HEIGHT: 1080
+      SE_SCREEN_DEPTH: 24
+      SE_SCREEN_DPI: 74
+    extra_hosts:
```

Comment:
An explicit IP address may cause problems in different environments. You never know what IP address/network will be assigned by the host systems.